src/core.jl

```diff
@@ -54,9 +54,13 @@ function single_ground_all_pairs(data::GraphData{T,V}, flags, cfg, log = true) w
         csinfo("Solver used: AMG accelerated by CG")
         amg_solver_path(data, flags, cfg, log)
     else
-        csinfo("Solver used: CHOLMOD")
+        s = cfg["solver"]
+        mklpardiso = false
+        str = s in CHOLMOD ? "CHOLMOD" : "MKL Pardiso"
+        s in MKLPARDISO && (mklpardiso = true)
+        csinfo("Solver used: $str")
         bs = parse(Int, cfg["cholmod_batch_size"])
-        _cholmod_solver_path(data, flags, cfg, log, bs)
+        cholmod_solver_path(data, flags, cfg, log, bs, mklpardiso)
```

Comment:
The function name is a bit strange. Can we use dispatch to pick the right one?

Reply:
Latest commit reorgs the code